src/blocktree.rs

```diff
@@ -1303,20 +1332,30 @@ pub fn create_tmp_sample_ledger(
     num_extra_ticks: u64,
     bootstrap_leader_id: Pubkey,
     bootstrap_leader_tokens: u64,
-) -> (Keypair, String, u64, Hash, Hash) {
+    ticks_per_slot: u64,
+) -> (Keypair, String, u64, u64, Hash, Hash) {
     let (genesis_block, mint_keypair) =
         GenesisBlock::new_with_leader(num_tokens, bootstrap_leader_id, bootstrap_leader_tokens);
     let ledger_path = get_tmp_ledger_path(name);
-    let (mut entry_height, mut last_entry_id) =
+    let (mut entry_height, mut tick_height, mut last_entry_id) =
         create_new_ledger(&ledger_path, &genesis_block).unwrap();
 
     let mut last_id = genesis_block.last_id();
     if num_extra_ticks > 0 {
         let entries = crate::entry::create_ticks(num_extra_ticks, last_entry_id);
 
         let blocktree = Blocktree::open(&ledger_path).unwrap();
+
+        //create_new_ledger creates one beginning tick
```

Comment:
nit: can you please leave one space between the // and the first letter of the comment.  It reads much better that way.